core/domain/stats_jobs_one_off_test.py

```diff
@@ -156,7 +156,8 @@ def setUp(self):
         self.exp1 = exp_services.get_exploration_by_id(self.exp1.id)
 
     def test_stats_models_regeneration_works(self):
-        """Test that stats models are regenerated with correct v1 stats values.
```

Comment:
I'm a bit confused. Why was this changed?

Even if the line is too long (and I'm not sure it is), wouldn't "stats" fit on the previous line?

Reply:
Done